WordPress/Classes/ViewRelated/NUX/NUXAbstractViewController.swift

```diff
@@ -158,10 +160,23 @@ class NUXAbstractViewController : UIViewController
     }
 
 
-    /// It is assumed that NUX view controllers are always presented modally. 
-    /// This method dismisses the view controller.
+    /// It is assumed that NUX view controllers are always presented modally.
+    /// This method dismisses the view controller by calling dismiss(cancelled:)
```

Comment:
I don't think the documentation should refer to a private method. Instead, refer to `dismissBlock`